### DISCUSSION TITLE
[2기-A]이창현

### DISCUSSION THREAD

**Discussion by changhyeonlee-0703** [super-corini/programmers_kdt_II#351] [2기 Monthly Project I]
### git : https://github.com/changhyeonlee-0703/django-blog

 1. 1기 netflix EDA하신 분 것을 많이 참고해봤는데... 어렵습니다. 역량을 한참 초과한 정도..ㅎ
 1. 우선 서버 배포 (aws -> pythonanywhere)를 중점으로 다뤘습니다.(django도 html, css도 처음한거라.. 제일 어려워보이는 것부터)
    -  처음에는 aws를 쓰다가 자꾸 대쉬보드가 꺼지는데... 이유를 모르겠네요..
    -  그래서 pythonanywhere로 넘어가서  배포하고자 했습니다.
 2. pythonanywhere 에선 
```
      DisallowedHost at /
      Invalid HTTP_HOST header: ~~~~~~~ to ALLOWED_HOSTS 
```  
   **settings.py**의 `ALLOWED_HOSTS = []` -> `ALLOWED_HOSTS = ['*']` 로 해결하였더니 금방 풀렸습니다.

이제 css를 해볼까합니다.

우선은 배포에 의의를 둡니다.(정신승리..)

**Comment by super-corini** [comment]
DEBUG 모드에서는 ALLOWED_HOST에 자동적으로 localhost가 들어가는데에 비해, 배포단에선 이것이 이루어지지 않아 발생하는 문제입니다. 잘 해결해주셨네요! 👍
